### PR TITLE
I have created my nevbar and footer

### PR DESCRIPTION
**Pull request by Farazbhai**

### CODE REVIEW

File: Layouts/footer.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<div class="bgm container-fluid mt-5">
+        <div class="row">
+            <div class="col-sm col-md-3">
+                <span class="mt-5 fw-bold text-info">CONNECT</span>
+                <p class="mt-3 text-light">2005 Stokes Isle Apt. 896,</p>
+                <p class="text-light">2005 Stokes Isle Apt. 896,</p>
+                <p class="mt-3 text-light">info@yourdomain.com</p>
+                <p class="mt-3 text-warning">(+68) 120034509</p>
+
+            </div>
```
Comment (Akram-baloch) [inline Layouts/footer.php:10]: remove this empty line

File: Layouts/footer.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="col-sm col-md-3">
+                <span class="mt-5 fw-bold text-info">EXTRA LINKS</span>
+                <p class="mt-3 text-light">Affiliate Program</p>
+                <p class="text-light">Business Services</p>
+                <p class="mt-3 text-light">Education Services</p>
+                <p class="mt-3 text-light">Gift Cards</p>
+
+            </div>
```
Comment (Akram-baloch) [inline Layouts/footer.php:18]: remove this empty line

File: Layouts/footer.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="col-sm col-md-3">
+                <span class="mt-5 fw-bold text-info">YOUR ACCOUNT</span>
+                <p class="mt-3 text-light">Shop</p>
+                <p class="text-light">My Orders</p>
+                <p class="mt-3 text-light">Gift Collections</p>
+                <p class="mt-3 text-light">Purchase Cards</p>
+                <span class="mt3 text-light">FAQs</span>
+
+            </div>
```
Comment (Akram-baloch) [inline Layouts/footer.php:27]: remove this empty line

File: Layouts/footer.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="col-sm col-md-3">
+                <span class="mt-5 fw-bold text-info">BEST SELLERS</span>
+                <div class="d-flex">
+                    <img class="w-25 mt-2" src="Assats/image/9.jpg" alt="">
```
Comment (Akram-baloch) [inline Layouts/footer.php:31]: fill image not found in alt

File: Layouts/footer.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                        <p class="text-light">$49.00 – $60.00</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container-fluid bg-black">
+        <div class="row">
+            <div class="col-sm col-lg-6">
+                <img src="Assats/image/11.jpg" alt="">
```
Comment (Akram-baloch) [inline Layouts/footer.php:43]: fill image not found in alt

File: Layouts/footer.php [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                <img src="Assats/image/10.jpg" alt="">
```
Comment (Akram-baloch) [inline Layouts/footer.php:44]: fill image not found in alt

File: Layouts/footer.php [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+                <img src="Assats/image/12.jpg" alt="">
```
Comment (Akram-baloch) [inline Layouts/footer.php:45]: fill image not found in alt

File: Layouts/footer.php [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+                <img src="Assats/image/13.jpg" alt="">
```
Comment (Akram-baloch) [inline Layouts/footer.php:46]: fill image not found in alt

File: Layouts/navbar.php [hunk 9]
```diff
@@ -0,0 +1,52 @@
+<div class="container mt-5">
+        <div class="row">
+            <div class="col-lg-1">
+                <i class="fa fa-bars mx-3" aria-hidden="true"></i>
+            </div>
+            <div class="col-lg-1">
+                <h6 class="mx-3">HOMES</h6>
+            </div>
+            <div class="col-lg-1">
+                <a class=" text-dark text-decoration-none " href="features.html">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:10]: remove feature .html in href

File: Layouts/navbar.php [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+                    <h6 class="mx-3"><a class="text-decoration-none text-dark" href="Feature.html"> FEATURES</a></h6>
+                </a>
+            </div>
+            <div class="col-lg-1">
+                <a class=" text-dark text-decoration-none " href="pages.html">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:15]: remove pages .html in href

File: Layouts/navbar.php [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+                    <h6 class="mx-3"><a class="text-decoration-none text-dark" href="Pages.html"> PAGES</a></h6>
```
Comment (Akram-baloch) [inline Layouts/navbar.php:16]: remove pages .html in href

File: Layouts/navbar.php [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+                </a>
+            </div>
+            <div class="col-lg-2">
+                <img class="mx-4" src="../assets/images/logo-dark.png" width="150px" alt="">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:20]: add image not found in alt

File: Layouts/navbar.php [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+            </div>
+            <div class="col-lg-1">
+                <a class=" text-dark text-decoration-none " href="shop.html">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:23]: remove shop .html in href

File: Layouts/navbar.php [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+                    <h6 class="mx-3">SHOP</h6>
+                </a>
+            </div>
+            <div class="col-lg-1">
+                <a class=" text-dark text-decoration-none " href="">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:28]: add # in href

File: Layouts/navbar.php [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+                    <h6 class="mx-3">EVENT</h6>
+                </a>
+            </div>
+            <div class="col-lg-1">
+                <a class=" text-dark text-decoration-none " href="blog.html">
```
Comment (Akram-baloch) [inline Layouts/navbar.php:33]: remove blog.html in href

File: assets/style/style.css [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+.hhh:hover {
+    color: red;
+}
+
+.border {
+    border: 2px solid grey;
+}
+
```
Comment (Akram-baloch) [inline assets/style/style.css:92]: format your code

**Review by Farazbhai — COMMENTED** [review]
I changed

File: Layouts/header.php [hunk 17]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (MuslimJaan) [inline Layouts/header.php:3]: remove empty line

File: Layouts/header.php [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="Assats/style.css">
+    <title>Bookstore
+    </title>
+</head>
+
```
Comment (MuslimJaan) [inline Layouts/header.php:15]: remove empty line

File: Layouts/navbar.php [hunk 19]
```diff
@@ -0,0 +1,52 @@
...
+                    <h6 class="mx-3">BLOG</h6>
+                </a>
+            </div>
+
```
Comment (MuslimJaan) [inline Layouts/navbar.php:37]: remove empty line

File: Layouts/navbar.php [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+            <div class="col-lg-1">
+                <i class="fa fa-search mx-3 " aria-hidden="true"></i>
+
```
Comment (MuslimJaan) [inline Layouts/navbar.php:40]: remove empty line

**Review by MuslimJaan — APPROVED** [review]

**Comment by Akram-baloch** [comment]
you are done

**Pull request merged**